src/assets/style/ShortcutGuideStyle.css

```
@@ -4,101 +4,138 @@
  * You should have received a copy of the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
  * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
  */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
 
-/* The Modal (background) */
-.a11y_modal {
-	display: none; /* Hidden by default */
-	position: fixed; /* Stay in place */
-	z-index: 1000; /* Placed on top */
+.shortcut-guide__background {
+	display: block;
+	position: fixed;
+	z-index: 1000;
 	left: 0;
 	top: 0;
-	width: 100%; /* Full width */
-	height: 100%; /* Full height */
-	overflow: auto; /* Enable scroll if needed */
-	background-color: rgb(0, 0, 0); /* Fallback color */
-	background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
-	font-family: "Open Sans", sans-serif;
-	font-size: small;
-	color: #f5f5ff;
-}
-
-.a11y_no_scroll {
-	overflow: hidden; /* disable scrolling on the main content */
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgba(0, 15, 58, 0.35);
+	color: #000f3a;
 }
 
-/* Modal Content/Box */
-.a11y_modal_content {
-	border-radius: 25px;
-	margin: 7% auto; /* 15% from the top and centered */
-	padding: 1rem;
-	padding-right: 2rem;
-	padding-left: 2rem;
-	border: 1px solid #888;
-	width: 80%; /* Could be more or less, depending on screen size */
-	border-radius: 21px;
-	background: rgba(0, 0, 0, 0.77);
+.shortcut-guide__container {
+	font-family: "Roboto";
```

We need a safe fallback font stack:
```suggestion
	font-family: "Roboto", Roboto, -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif;
```

This loads the text much faster, as there is no [flash of unstyled text](https://css-tricks.com/fout-foit-foft/)